src/tests/system/tests/test_ldap.py

```diff
@@ -492,19 +427,107 @@ def test_ldap__password_change_no_grace_logins_left(
         2. With 'exop' expect just a failed login
     :customerscenario: False
     """
-    ldap.ldap.modify("cn=config", replace={"passwordExp": "on", "passwordMaxAge": "1", "passwordGraceLimit": "0"})
-    ldap.user("user1").add(password="Secret123")
-
-    # make sure the password is expired
-    time.sleep(3)
+    ldap.ldap.modify("cn=config", replace={"passwordMaxAge": "1", "passwordGraceLimit": "0"})
+    ldap.user("user1").add(password="Secret123").password_change_at_logon(password="Secret123")
 
     client.sssd.domain["ldap_pwmodify_mode"] = modify_mode
     client.sssd.start()
 
-    rc, _, _, _ = client.auth.parametrize(method).password_with_output("user1", "Secret123")
+    rc, _, _, _ = client.auth.ssh.password_with_output("user1", "Secret123")
     assert rc == expected, err_msg
 
 
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_ldap__empty_attribute(client: Client, ldap: LDAP):
```

Comment:
These tests were not part of the commit being back-ported.

Reply:
This per se doesn't mean it shouldn't.
Does the test make sense in sssd-2-9?

Reply:
I removed the tests.